enterprise/coderd/provisionerdaemons_test.go

```diff
@@ -397,10 +398,10 @@ func TestProvisionerDaemonServe(t *testing.T) {
 				Tags: map[string]string{
 					provisionersdk.TagScope: provisionersdk.ScopeOrganization,
 				},
-				PreSharedKey: "provisionersftw",
+				PreSharedKey: provPSK,
 			})
 		}, &provisionerd.Options{
-			Logger:    logger.Named("provisionerd"),
+			Logger:    logger.Named(provPSK),
```

Comment:
Why is the logger changed to the PSK? I'm assuming over-zealous replacement of strings? This is only a test, but yikes if we ever did that in production!

Reply:
Over-zealous...

Reply:
Good catch